static/dependencies.config.php

```diff
@@ -19,266 +20,237 @@
 	],
-	ICanCreateInstances::class   => [
-		'instanceOf' => DiceInstanceManager::class,
+	\Friendica\Core\Hooks\Capability\ICanCreateInstances::class => [
+		'instanceOf' => \Friendica\Core\Hooks\Model\DiceInstanceManager::class,
 		'constructParams' => [
 			[Dice::INSTANCE => Dice::SELF],
 		],
 	],
-	Config\Util\ConfigFileManager::class => [
-		'instanceOf' => Config\Factory\Config::class,
-		'call'       => [
+	\Friendica\Core\Config\Util\ConfigFileManager::class => [
+		'instanceOf' => \Friendica\Core\Config\Factory\Config::class,
+		'call' => [
 			['createConfigFileManager', [
-				[Dice::INSTANCE => '$basepath'],
-				$_SERVER,
+				$basepath,
+				$serverVars,
 			], Dice::CHAIN_CALL],
 		],
 	],
-	Config\ValueObject\Cache::class => [
-		'instanceOf' => Config\Factory\Config::class,
-		'call'       => [
+	\Friendica\Core\Config\ValueObject\Cache::class => [
+		'instanceOf' => \Friendica\Core\Config\Factory\Config::class,
+		'call' => [
 			['createCache', [], Dice::CHAIN_CALL],
 		],
 	],
-	App\Mode::class              => [
+	\Friendica\App\Mode::class => [
 		'call' => [
-			['determineRunMode', [true, $_SERVER], Dice::CHAIN_CALL],
+			['determineRunMode', [true, $serverVars], Dice::CHAIN_CALL],
 			['determine', [
-				[Dice::INSTANCE => '$basepath']
+				$basepath,
 			], Dice::CHAIN_CALL],
 		],
 	],
-	Config\Capability\IManageConfigValues::class => [
-		'instanceOf' => Config\Model\DatabaseConfig::class,
+	\Friendica\Core\Config\Capability\IManageConfigValues::class => [
+		'instanceOf' => \Friendica\Core\Config\Model\DatabaseConfig::class,
 		'constructParams' => [
-			$_SERVER,
+			$serverVars,
 		],
 	],
-	PConfig\Capability\IManagePersonalConfigValues::class => [
-		'instanceOf' => PConfig\Factory\PConfig::class,
-		'call'       => [
+	\Friendica\Core\PConfig\Capability\IManagePersonalConfigValues::class => [
+		'instanceOf' => \Friendica\Core\PConfig\Factory\PConfig::class,
+		'call' => [
 			['create', [], Dice::CHAIN_CALL],
 		]
 	],
-	DbaDefinition::class => [
+	\Friendica\Database\Definition\DbaDefinition::class => [
 		'constructParams' => [
-			[Dice::INSTANCE => '$basepath'],
+			$basepath,
 		],
 		'call' => [
 			['load', [false], Dice::CHAIN_CALL],
 		],
 	],
-	ViewDefinition::class => [
+	\Friendica\Database\Definition\ViewDefinition::class => [
 		'constructParams' => [
-			[Dice::INSTANCE => '$basepath'],
+			$basepath,
 		],
 		'call' => [
 			['load', [false], Dice::CHAIN_CALL],
 		],
 	],
-	Database::class                         => [
+	\Friendica\Database\Database::class => [
 		'constructParams' => [
-			[Dice::INSTANCE => Config\Model\ReadOnlyFileConfig::class],
+			[Dice::INSTANCE => \Friendica\Core\Config\Model\ReadOnlyFileConfig::class],
 		],
 	],
 	/**
-	 * Creates the App\BaseURL
+	 * Creates the \Friendica\App\BaseURL
 	 *
 	 * Same as:
-	 *   $baseURL = new App\BaseURL($configuration, $_SERVER);
+	 *   $baseURL = new \Friendica\App\BaseURL($configuration, $);
```

Comment:
The second parameter of the constructor of `BaseURL` is empty in this line. Is this intended?

Reply:
Good catch. I removed the complete comment because it didn't add any value.